Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackWidget.swift

```diff
@@ -151,6 +58,57 @@ struct SentryUserFeedbackWidget {
             return result
         }
     }
+
+    func setWidget(visible: Bool, animated: Bool) {
+        rootVC.setWidget(visible: visible, animated: animated)
+    }
+
+    class RootViewController: UIViewController, UIAdaptivePresentationControllerDelegate {
+        let defaultWidgetSpacing: CGFloat = 8
+        unowned let button: SentryUserFeedbackWidgetButtonView
```

Comment:
`m`: Why `unowned` and not weak? While we are in control in the reference, it is easy to break unowned references, which can result in SDK crashes. Using `weak` with nil-unwrapping would be safer to use.

Reply:
I'll use the weak ref